### PR TITLE
[inner-1929] adjust reload log

### PR DESCRIPTION
**Pull request by wenyh1**
Reason:  
  BUG inner1929
Type:  
  BUG/Improve  
Influences：  
   fix xx

### CODE REVIEW

File: src/main/java/com/actiontech/dble/meta/ReloadLogHelper.java [hunk 1]
```diff
@@ -28,96 +28,64 @@ public ReloadLogHelper(boolean isReload) {
...
 
-    public void warn(String message) {
-        LOGGER.warn(getStage() + message);
+    public static void info2(String message) {
+        LOGGER.info(getLogStage() + message);
     }
 
-    public void warn(String message, Throwable var2) {
-        LOGGER.warn(getStage() + message, var2);
+    public static void info3(String message) {
```
Comment (PanternBao) [inline src/main/java/com/actiontech/dble/meta/ReloadLogHelper.java:71]: info2 and info3 need straightforward method name

File: src/main/java/com/actiontech/dble/services/manager/handler/UpdateHandler.java [hunk 2]
```diff
@@ -189,12 +194,12 @@ public ManagerWritableTable getWritableTable(MySqlUpdateStatement update, Manage
         return (ManagerWritableTable) managerBaseTable;
     }
 
-    private int updateRows(ManagerService service, ManagerWritableTable managerTable, Set<LinkedHashMap<String, String>> affectPks, LinkedHashMap<String, String> values, PacketResult packetResult) throws Exception {
```
Comment (PanternBao) [inline src/main/java/com/actiontech/dble/services/manager/handler/UpdateHandler.java:192]: why remove packet result?

File: src/main/java/com/actiontech/dble/meta/ReloadLogHelper.java (hunk 1 above)
Comment (wenyh1) [reply src/main/java/com/actiontech/dble/meta/ReloadLogHelper.java:71]: get

File: src/main/java/com/actiontech/dble/services/manager/handler/UpdateHandler.java (hunk 2 above)
Comment (wenyh1) [reply src/main/java/com/actiontech/dble/services/manager/handler/UpdateHandler.java:192]: use ReloadException.java, uniform capture at the outermost layer method

**Comment by wenyh1** [comment]
Because of conflicting files, too many, restart one: #3491

**Pull request closed without merging**